pivot/static/pivot/js/major.js

```diff
@@ -377,14 +434,14 @@ function displayResults() {
                     status: checked,
                     data: _completeMajorMap[maj]["major_full_nm"]
                 }));
-                $(appendTo + " li:last").data("code", maj);
+                $(appendTo + " label:last").data("code", maj);
                 count++;
             }
         }
     }
 
     //start timer to make suggestions box disappear after 1sec
-    clearTimeout(_timer);
+    //clearTimeout(_timer);
```

Comment:
Let's double check to see if this timer and clearTimeout should be removed permanently.